src/com/company/Main.java

```diff
@@ -35,153 +33,62 @@ public static void play() {
-            //sprawdzenie
-            shouldPlayAgain = verifyIfContinue(boardSize, board);
-
-            //ruch "O"
-            //drawField(boardSize,board);
-
-
         } while (shouldPlayAgain);
         System.out.println(END_OF_THE_GAME);
     }
 
-    private static boolean putToBoard(final int row, final int column, final PlayerSignType sign, final Board board) {
-        if (!board.insertSign(sign, row - 1, column - 1)) {
-            System.out.println(NOT_EMPTY_PLACE);
-            return false;
-        }
-        return true;
-    }
-
-    private static boolean verifyIfContinue(final int boardSize, final Board board) {
-        return verifyIfRowIsNotFull(boardSize, board)
-                && verifyIfColumnIsNotFull(boardSize, board)
-                && verifyIfDiagonalXxIsNotFull(boardSize, board)
-                && verifyIfDiagonalYyIsNotFull(boardSize, board);
-    }
-
-    public static boolean verifyIfDiagonalYyIsNotFull(final int boardSize, final Board board) {
-        int intForCheck = board.countDiagonalYyValue();
-
-        if (abs(intForCheck) == boardSize) {
-            announceTheWinner(intForCheck, DIAGONAL_JJ, 2);  //jak numerować przekątne?
-            return false;
-        }
-        return true;
-
-    }
-
-    public static boolean verifyIfDiagonalXxIsNotFull(final int boardSize, final Board board) {
-        int intForCheck = board.countDiagonalXxValue();
-
-        if (abs(intForCheck) == boardSize) {
-            announceTheWinner(intForCheck, DIAGONAL_II, 1);  //jak numerować przekątne?
-            return false;
-        }
-        return true;
-
-    }
-
-    public static boolean verifyIfColumnIsNotFull(final int boardSize, final Board board) {
-        final int[] tabForCheck = new int[boardSize];
-        for (int i = 0; i < tabForCheck.length; i++) {
-            tabForCheck[i] = 0;
-        }
-
-        for (int i = 0; i < boardSize; i++) {
-            for (int j = 0; j < boardSize; j++) {
-                if (board.isSignEqual(PlayerSignType.X, j, i)) {
-                    tabForCheck[i] += 1;
-                } else {
-                    if (board.isSignEqual(PlayerSignType.O, j, i)) {
-                        tabForCheck[i] -= 1;
-                    }
-                }
-            }
-        }
-
-        for (int i = 0; i < tabForCheck.length; i++) {
-            if (abs(tabForCheck[i]) == boardSize) {
-                announceTheWinner(tabForCheck[i], COLUMN, (i + 1));
-                return false;
-            }
-        }
-        return true;
-    }
-
-    public static boolean verifyIfRowIsNotFull(final int boardSize, final Board board) {
-        final int[] tabForCheck = new int[boardSize];
-        for (int i = 0; i < tabForCheck.length; i++) {
-            tabForCheck[i] = 0;
+    public static PlayerSignType changePlayer(PlayerSignType player) {
+        if (player == PlayerSignType.O) {
+            player = PlayerSignType.X;
+        } else if (player == PlayerSignType.X) {
```

Comment:
To sprawdzanie `if (player == PlayerSignType.X)` jest chyba nie potrzebne bo skoro wiemy że nie jest kółkiem, to musi być X-sem, nie? A nawet jeśli byłby np EMPTY to i tak chcemy z niego zrobić kółko.
Czyli sam `} else {` by wystaczył.

Reply:
Poprawione. Empty nie powinno być nigdy. Jedyne nad czym się zastanawiam to moment kiedy doszłoby do zmiany gracza mimo że nie było wykonanego ruchu (błędne dane, poza zakresem)